pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -182,26 +235,25 @@ def _get_wrapped_positions(self, positions, distance_buffer=1.0e-12):
         x = np.array(positions).copy()
         cell = self._ref_structure.cell
         x_scale = np.dot(x, np.linalg.inv(cell))+distance_buffer
-        x[...,self._ref_structure.pbc] -= np.dot(np.floor(x_scale),
-                                                 cell)[...,self._ref_structure.pbc]
+        x[...,self._ref_structure.pbc] -= np.dot(
+            np.floor(x_scale), cell
+        )[...,self._ref_structure.pbc]
         return x
 
+    @deprecate(allow_ragged="use mode instead.")
```

Comment:
Do we really need to deprecate private functions? I lean towards just removing the `allow_ragged` here!

Reply:
The reason why `allow_ragged` is still here is because `_get_distances_and_indices` is used by multiple functions -> by leaving `allow_ragged` here, the conversion of `allow_ragged` to `mode` has to be done only once.

But actually you might be right that maybe it's not necessary to have `@deprecate` here, since the user should be able to see the warning in an earlier function.